### PR TITLE
Add option to show and unignore ignored issues

### PR DESCRIPTION
**Pull request by rubenvanerk**
This PR adds the option to view ignored issues. It also enables you to un-ignore an ignored issue.

I removed the ignore modal since the action isn't as destructive as before.

image

### CODE REVIEW

**Comment by ash-jc-allen** [comment]
Hey @rubenvanerk! I love this!

The only thing that I think I'd change would be to move the toggle to the bottom of the side nav. My thinking behind it is that if someone's ignored an issue, it's probably not going to be that important for them to want to un-ignore it. But I'm definitely open to suggestions on it though; what are your thoughts on it? 😄

**Comment by rubenvanerk** [comment]
Thanks!

I do agree with you but discoverability of would be pretty low if the toggle is all the way at the bottom. I can imagine a scenario where someone accidentally ignores an issue and then not being able to find this feature.

I moved it below the sort function, let me know what you think:
image

File: resources/views/components/ignore-issue.blade.php [hunk 1]
```diff
@@ -1,80 +1,37 @@
...
+                 stroke="currentColor" stroke-width="2">
+                <path stroke-linecap="round" stroke-linejoin="round"
+                      d="M13.875 18.825A10.05 10.05 0 0112 19c-4.478 0-8.268-2.943-9.543-7a9.97 9.97 0 011.563-3.029m5.858.908a3 3 0 114.243 4.243M9.878 9.878l4.242 4.242M9.88 9.88l-3.29-3.29m7.532 7.532l3.29 3.29M3 3l3.59 3.59m0 0A9.953 9.953 0 0112 5c4.478 0 8.268 2.943 9.543 7a10.025 10.025 0 01-4.132 5.411m0 0L21 21"/>
+            </svg>
+        @endif
     </button>
 
-    <div x-show="ignoreModalOpen" class="relative z-10" aria-labelledby="modal-title"
-         role="dialog" aria-modal="true">
-        <div class="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity"></div>
```
Comment (wadakatu) [inline resources/views/components/ignore-issue.blade.php:27]: I think that the modal is still helpful for users who might misclick or mistap the ignored icon, even if there is a toggle option to change either show or hide ignored issues.

What do you think about it ? @ash-jc-allen @rubenvanerk 
Thank you.

Comment (rubenvanerk) [reply resources/views/components/ignore-issue.blade.php:27]: Alpine components has one: https://alpinejs.dev/component/notifications. I could implement that.

Comment (ash-jc-allen) [reply resources/views/components/ignore-issue.blade.php:27]: Nice! I've not got a license for Alpine Components, unfortunately. But, if you have one and it doesn't break any license rules, I'd be happy for you to add the notifications.

If you don't want to though, I'd be happy to add something myself :)

**Pull request merged**